app/models/UserDetails.scala

```diff
@@ -19,7 +19,6 @@ package models
 import play.api.libs.json.{Json, OFormat}
 
 final case class UserDetails(
-                             VATNumber: Option[VRN],
```

Comment:
@somaraja can we put isVatRegistered in here?

Reply:
@lmugnai this is incorporated